diff -ur exo-0.3.101.orig//exo/exo-string.c exo-0.3.101/exo/exo-string.c
--- exo-0.3.101.orig//exo/exo-string.c	2008-11-11 08:22:46.000000000 +0000
+++ exo-0.3.101/exo/exo-string.c	2011-12-13 11:11:20.613147984 +0000
@@ -400,7 +400,7 @@
  *
  * Since: 0.3.1.1
  */
-G_CONST_RETURN gchar*
+const gchar*
 exo_intern_string (const gchar *string)
 {
 #if GLIB_CHECK_VERSION(2,9,0)
@@ -425,7 +425,7 @@
  *
  * Since: 0.3.1.1
  */
-G_CONST_RETURN gchar*
+const gchar*
 exo_intern_static_string (const gchar *string)
 {
 #if GLIB_CHECK_VERSION(2,9,0)
diff -ur exo-0.3.101.orig//exo/exo-string.h exo-0.3.101/exo/exo-string.h
--- exo-0.3.101.orig//exo/exo-string.h	2008-11-11 08:22:46.000000000 +0000
+++ exo-0.3.101/exo/exo-string.h	2011-12-13 11:11:25.292147864 +0000
@@ -46,8 +46,8 @@
 gchar               **exo_strndupv               (gchar          **strv,
                                                   gint             num) G_GNUC_MALLOC G_GNUC_WARN_UNUSED_RESULT;
 
-G_CONST_RETURN gchar *exo_intern_string          (const gchar     *string);
-G_CONST_RETURN gchar *exo_intern_static_string   (const gchar     *string);
+const gchar *exo_intern_string          (const gchar     *string);
+const gchar *exo_intern_static_string   (const gchar     *string);
 
 #if GLIB_CHECK_VERSION(2,9,0)
 #define I_(string) (g_intern_static_string ((string)))
diff -ur exo-0.3.101.orig//exo-support/xfce-heading.c exo-0.3.101/exo-support/xfce-heading.c
--- exo-0.3.101.orig//exo-support/xfce-heading.c	2008-11-11 08:22:46.000000000 +0000
+++ exo-0.3.101/exo-support/xfce-heading.c	2011-12-13 11:10:58.620085167 +0000
@@ -609,7 +609,7 @@
  *
  * Since: 4.4.0
  **/
-G_CONST_RETURN gchar*
+const gchar*
 xfce_heading_get_icon_name (XfceHeading *heading)
 {
   g_return_val_if_fail (XFCE_IS_HEADING (heading), NULL);
@@ -662,7 +662,7 @@
  *
  * Since: 4.4.0
  **/
-G_CONST_RETURN gchar*
+const gchar*
 xfce_heading_get_subtitle (XfceHeading *heading)
 {
   g_return_val_if_fail (XFCE_IS_HEADING (heading), NULL);
@@ -713,7 +713,7 @@
  *
  * Since: 4.4.0
  **/
-G_CONST_RETURN gchar*
+const gchar*
 xfce_heading_get_title (XfceHeading *heading)
 {
   g_return_val_if_fail (XFCE_IS_HEADING (heading), NULL);
diff -ur exo-0.3.101.orig//exo-support/xfce-heading.h exo-0.3.101/exo-support/xfce-heading.h
--- exo-0.3.101.orig//exo-support/xfce-heading.h	2008-11-11 08:22:46.000000000 +0000
+++ exo-0.3.101/exo-support/xfce-heading.h	2011-12-13 11:11:14.733022668 +0000
@@ -65,15 +65,15 @@
 void                  xfce_heading_set_icon      (XfceHeading *heading,
                                                   GdkPixbuf   *icon);
 
-G_CONST_RETURN gchar *xfce_heading_get_icon_name (XfceHeading *heading);
+const gchar *xfce_heading_get_icon_name (XfceHeading *heading);
 void                  xfce_heading_set_icon_name (XfceHeading *heading,
                                                   const gchar *icon_name);
 
-G_CONST_RETURN gchar *xfce_heading_get_subtitle  (XfceHeading *heading);
+const gchar *xfce_heading_get_subtitle  (XfceHeading *heading);
 void                  xfce_heading_set_subtitle  (XfceHeading *heading,
                                                   const gchar *subtitle);
 
-G_CONST_RETURN gchar *xfce_heading_get_title     (XfceHeading *heading);
+const gchar *xfce_heading_get_title     (XfceHeading *heading);
 void                  xfce_heading_set_title     (XfceHeading *heading,
                                                   const gchar *title);
 
diff -ur exo-0.3.101.orig//exo-support/xfce-titled-dialog.c exo-0.3.101/exo-support/xfce-titled-dialog.c
--- exo-0.3.101.orig//exo-support/xfce-titled-dialog.c	2008-11-11 08:22:46.000000000 +0000
+++ exo-0.3.101/exo-support/xfce-titled-dialog.c	2011-12-13 11:11:07.323085371 +0000
@@ -328,7 +328,7 @@
  *
  * Since: 4.4.0
  **/
-G_CONST_RETURN gchar*
+const gchar*
 xfce_titled_dialog_get_subtitle (XfceTitledDialog *titled_dialog)
 {
   g_return_val_if_fail (XFCE_IS_TITLED_DIALOG (titled_dialog), NULL);
diff -ur exo-0.3.101.orig//exo-support/xfce-titled-dialog.h exo-0.3.101/exo-support/xfce-titled-dialog.h
--- exo-0.3.101.orig//exo-support/xfce-titled-dialog.h	2008-11-11 08:22:46.000000000 +0000
+++ exo-0.3.101/exo-support/xfce-titled-dialog.h	2011-12-13 11:11:02.684085193 +0000
@@ -66,7 +66,7 @@
                                                            const gchar      *first_button_text,
                                                            ...) G_GNUC_MALLOC;
 
-G_CONST_RETURN gchar *xfce_titled_dialog_get_subtitle     (XfceTitledDialog *titled_dialog);
+const gchar *xfce_titled_dialog_get_subtitle     (XfceTitledDialog *titled_dialog);
 void                  xfce_titled_dialog_set_subtitle     (XfceTitledDialog *titled_dialog,
                                                            const gchar      *subtitle);
 
